lib/tls.js

```diff
@@ -60,6 +63,9 @@ function convertProtocols(protocols) {
   const lens = new Array(protocols.length);
   const buff = Buffer.allocUnsafe(protocols.reduce((p, c, i) => {
     var len = Buffer.byteLength(c);
+    if (len > 255) {
+      throw new ERR_OUT_OF_RANGE('byte length of a protocol', '< 255', len);
```

Comment:
Should this be `<= 255` in the error message?

Reply:
Oops, you're right about that.